example/javascriptlib/basic/1-simple/build.mill

```diff
@@ -1,27 +1,36 @@
 package build
+
 import mill._, javascriptlib._
 
-object foo extends TypeScriptModule {
-  object bar extends TypeScriptModule {
-    def npmDeps = Seq("immutable@4.3.7")
-  }
-}
+object foo extends JestModule {
+  def npmDeps = Seq("immutable@4.3.7")
 
-object qux extends TypeScriptModule {
-  def moduleDeps = Seq(foo, foo.bar)
+  override def mainFilePath =
+    Task { compile()._1.path / millSourcePath.last / "src" / mainFileName() }
```

Comment:
You mentioned getting rid of `millSourcePath.last`, needs to be gotten rid of here as well

Reply:
yes, that has been done waiting on basic-1 to be merged then I will rebase and push updates